sklearn/metrics/scorer.py

```diff
@@ -313,12 +322,12 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
 
 # Standard regression scores
 r2_scorer = make_scorer(r2_score)
-mean_squared_error_scorer = make_scorer(mean_squared_error,
-                                        greater_is_better=False)
-mean_absolute_error_scorer = make_scorer(mean_absolute_error,
-                                         greater_is_better=False)
-median_absolute_error_scorer = make_scorer(median_absolute_error,
-                                           greater_is_better=False)
```

Comment:
Ah, it's an object, not a function, so it doesn't go into the right branch in deprecated, right? We could still wrap the "__call__" method, right?